org.mwc.debrief.core/src/org/mwc/debrief/core/ui/ImportNarrativeDialog.java

```diff
@@ -0,0 +1,112 @@
+/**
+ * 
+ */
+package org.mwc.debrief.core.ui;
+
+import org.eclipse.jface.dialogs.Dialog;
+import org.eclipse.swt.SWT;
+import org.eclipse.swt.events.SelectionAdapter;
+import org.eclipse.swt.events.SelectionEvent;
+import org.eclipse.swt.events.SelectionListener;
+import org.eclipse.swt.layout.GridData;
+import org.eclipse.swt.layout.GridLayout;
+import org.eclipse.swt.widgets.Button;
+import org.eclipse.swt.widgets.Composite;
+import org.eclipse.swt.widgets.Control;
+import org.eclipse.swt.widgets.Label;
+import org.eclipse.swt.widgets.Shell;
+
+import Debrief.ReaderWriter.Word.ImportNarrativeDocument.ImportNarrativeEnum;
+
+/**
+ * Dialog popped up from {@link ImportNarrativeHelper}
+ * @author Ayesha
+ *
+ */
+public class ImportNarrativeDialog extends Dialog
+{
+  private Button _btnLoadedTracks;
+  private Button _btnAllData;

```

Comment:
@ayeshare - the code checker is indicating that _btnAllData could have been a variable defined in the createDialogArea method - since it is only referred to in that method.